### DISCUSSION TITLE
[Bug]: Can't press select while in game on pc version

### DISCUSSION THREAD

**Discussion by Redivh** [visualboyadvance-m/visualboyadvance-m#1155] [General]
### Expected behavior?

Be able to press select while paying a game.

### Actual behavior?

I can't press select in game, the key is set to backspace but when I press it nothing happens. I also tried to reconfigure the input to another key but it still won't work.

### Steps to reproduce the problem

1. Open emulator
2. Open GBC Rom file
3. Select game file
4. Let the game load
5. Wait until you need to press select in the game
6. The select key input won't work

### Error messages?

If this is a crash bug, and you have the time and inclination, please provide a symbolic backtrace.
Instructions for doing so are [here](https://github.com/visualboyadvance-m/visualboyadvance-m/blob/master/README.md#reporting-crash-bugs).

### What game were you playing

Game Titles:
Donkey Kong 1994, Super Mario Bros Deluxe

### What settings are you using?

Default Factory Settings

### Specifications and Build details

VBA-M: (include commit version number and whether you are using the 32-bit or 64-bit version) 2.1.6 64-bit
OS: (include version and OS build number using WinVer if on Windows): Windows

### Copyright Compliance

- [X] There are no infringing files attached to this bug report.

**Comment by Squall-Leonhart** [comment]
Backspace is bound to Turbo  by default, which overrides your select binding,

**Reply by Redivh** [reply]
How do I turn that off?

**Reply by Squall-Leonhart** [reply]
rebind it on the special tab of the input config.
